quactography/solver/qaoa_solver.py

```diff
@@ -0,0 +1,84 @@
+from qiskit.primitives import Estimator, Sampler
```

Comment:
I would call this file `qaoa.py` simply. We might add other types of solver (so new files) someday.